### ISSUE TITLE
escape variable

### ISSUE THREAD

**Issue by m-tmatma** [m-tmatma/CreateGUIDVSPlugin#44]
escape variable

```
{{LowerHyphens}}   => {LowerHyphens}
{{{LowerHyphens}}} => {8a9667fa-a179-400c-b07e-03c1142ba76c}

```

**Comment by m-tmatma** [comment]
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/exception

**Comment by m-tmatma** [comment]
merged #46
